vue/src/components/SideBar.vue

```diff
@@ -159,8 +160,10 @@ const satelliteLoadingColor = computed(() => {
   return 'black'
 })
 
-
-
+const refreshListings = () => {
+  updatePerformers();
+  updateRegionList();
```

Comment:
How about merging `updateRegionMap` into `updateRegionList`? I see that these two functions are always called together, so I can move `updateRegionMap` into `updateRegionList`, keeping the name `updateRegionList`.

Reply:
Go for it, if they are always called together that should be fine.  I really don't remember why I separated them out.  I had something to do with custom Regions but it may have been just for that PR.